E2E-tests/tests/test_smoke.py

```diff
@@ -3,10 +3,37 @@
 from src.blockchain_api import BlockchainApi, Wallet, Transaction
 from config.api_config import ApiConfig
 import logging as logger
+import json
 
 
 @mark.CD
 class TestSmoke:
+    @mark.substrate
+    def test_cardano_nodes_are_in_sync(self, api: BlockchainApi):
```

Comment:
You can't test cardano network being in sync looking at tips.

Nodes are not forked if their stable blocks match.
The check is that:
* query nodes for their newest stable block (securityParam blocks below the tip)
* select lowest (earliest) block number from them
* compare block with this number among all cardano nodes